sequencer.c

```diff
@@ -2145,6 +2153,22 @@ static int do_pick_commit(struct repository *r,
 		if (res || command != TODO_REWORD)
 			goto leave;
 		reword = 1;
+		if (strlen(msg.subject) != item->arg_len ||
+		    strncmp(msg.subject, item->arg_offset,
+			    item->arg_len) != 0) {
+			const char *commit_buf, *subject;
```

Comment:
Git's coding convention does not allow declarations after statements. The declarations of variables must come first.

Reply:
I'm not sure I understand your point.
First line of this block there are declarations only.
What am I missing ?